plugins/chat/spec/system/page_objects/chat/chat_thread.rb

```diff
@@ -3,6 +3,10 @@
 module PageObjects
   module Pages
     class ChatThread < PageObjects::Pages::Base
+      def open?
+        has_css?(".chat-thread:not(.loading)")
```

Comment:
I think we should include a `loaded` state instead of `:not(.loading)`. Depending on how fast playwright is, this assertion may run before the browser has rendered the loading state and therefore pass.